stores/astra/src/vector/index.test.ts

```diff
@@ -3,7 +3,7 @@ import { vi, describe, it, expect, beforeAll, afterAll, test, beforeEach, afterE
 import { AstraVector } from './';
 
 // Give tests enough time to complete database operations
-vi.setConfig({ testTimeout: 3000000 });
+vi.setConfig({ testTimeout: 3000000, hookTimeout: 3000000 });
```

Comment:
should probably go for 300000 since that's five mins - I think what you have might be five hours haha